docs/user-guide/profile.md

```diff
@@ -155,14 +159,14 @@ readable form.
     ||  13.4% | 113.9 |  26.1 | 18.8% | __cray_memcpy_SNB
     |==================================================
 
-This report will generate two more files, one with the extension `.ap2`
-which holds the same data as the `.xf` but in the post processed form.
-The other file has a `.apa` extension and is a text file with a
-suggested configuration for generating a traced experiment. The `.ap2`
-file generated is used to view performance data graphically with the
-Cray Apprentice2 tool (for information on using this, please take a look
-at [Cray Apprentice2](#cray-apprentice2)), and the latter is used for
-more detailed tracing experiments.
+This report will generate more files with the extension `.ap2` in the
+experiment directory. These hold the same data as the `.xf` file but
+in the post-processed form. Another new file has a `.apa` extension
+and is a text file with a suggested configuration for generating a
+traced experiment. The `.ap2` files generated are used to view performance
+data graphically with the Cray Apprentice2 tool (for information on using
+this, please take a look at [Cray Apprentice2](#cray-apprentice2)), and
```

Comment:
(#cray-apprentice2)) has two ")"

Reply:
I spotted that too, first time around! I think it's ok -- one ')' is to close `(#cray-apprentice2)` for the link to the Apprentice2 documentation, and the other closes '(for information on using...'.